vignettes/package_structure.Rmd

```diff
@@ -22,124 +22,180 @@ To make it easier for new developers to join, this documentation is created.
 
 # Package Structures
 
-`mmrm` is an R package, and the following sections describe all files/folders that are contained in this package.
+`mmrm` is an R package, and the following sections describe all files/folders that are contained in this package as well as the repository.
 
-## data
+## `cache`
 
-The `data/` folder is where `mmrm` internal data is stored.
-See the [`fev_data` documentation](https://openpharma.github.io/mmrm/main/reference/fev_data.html) and the [introduction vignette](introduction.html#common-usage) for details.
+This folder contains cache files, and is not part of the package.
```

Comment:
is this `cache` folder still needed? (if it is for the vignette, it can be removed I think)